frontend/src/Components/SignUp/SignUp.js

```diff
@@ -12,6 +13,20 @@ const SignUp = ({ history }) => {
         await app
           .auth()
           .createUserWithEmailAndPassword(email.value, password.value);
+        ////////////////////////////////////////
```

Comment:
You are using twice the same code, maybe extract it to some kind of `util` file and import it here?